model/transaction/_meta/fields.yml

```diff
@@ -16,6 +16,17 @@
               description: >
                 Total duration of this transaction, in microseconds.
 
+            - name: count
```

Comment:
so these fields (and others) correspond to the `transaction` type as it comes in the `metricsets`, but they are not passed in the `transaction` events.  is this the right place to put the mapping?

Reply:
i think only `breakdown.count` is strictly required by UI now, so I can remove the others

Reply:
I'd either define it in the `_meta/fields.common.yml` or introduce a `transaction`  namespace in the `metricset` package. It shouldn't really change anything, as in the end it's all combined in the `fields.yml`, but I think it is easier to find again.